docs/en/setup/backend/backend-setup.md

```diff
@@ -115,3 +115,15 @@ which format process is timezone related.
   
 In default, SkyWalking OAP backend choose the OS default timezone.
 If you want to override it, please follow Java and OS documents to do so.
+
+#### How to query the storage directly from 3rd party tool?
+SkyWalking provides browser UI, CLI and GraphQL ways to support extensions. But some users may have the idea to query data 
+directly from the storage. Such as in ElasticSearch case, Kibana is a great tool to do this.
+
+In default, due to reduce memory, network and storage space usages, SkyWalking saves id(s) only in the entity and metadata saved in the
+*_inventory entities only. But these tools usually don't support nested query, or don't work conveniently. In this special case,
```

Comment:
The italic font style is not terminated?

Reply:
No, it isn't. How to force it back in the markdown?

Reply:
`*` and `_` should appear in pair IMO

Reply:
Fixed.